love/src/components/MainTel/M1M3/M1M3.jsx

```diff
@@ -330,7 +330,7 @@ export default class M1M3 extends Component {
 
     if (
       this.state.selectedForceParameter !== prevState.selectedForceParameter ||
-      this.props[this.state.selectedForceInput] !== prevProps[this.state.selectedForceInput]
+      !_.isEqual(this.props[this.state.selectedForceInput], prevProps[this.state.selectedForceInput])
```

Comment:
This nomenclature is new to me, the "!_.isEqual(...)"

Reply:
Good observation! The `_` is the `lodash` library, on the documentation examples the use the `_` but this could lead to confusions I think. Perhaps we should probably use `lodash` instead. I'll make the change here.
About the `isEqual` method: it used to make a deep comparison of objects and this is needed when comparing `Object` variables on the `componentDidUpdate` method, so we can make sure they are really different (using A == B where A and B are objects won't work even do they have the same content, because with `==` we are checking its pointer, nor its value)

Reply:
Thanks for the explanation!